src/testrender/shading.cpp

```diff
@@ -1446,6 +1452,113 @@ struct MxSheen final : public BSDF, MxSheenParams {
     }
 };
 
+// Implement the sheen model proposed in:
+//  "Practical Multiple-Scattering Sheen Using Linearly Transformed Cosines"
+// Tizian Zeltner, Brent Burley, Matt Jen-Yuan Chiang - Siggraph 2022
+// https://tizianzeltner.com/projects/Zeltner2022Practical/
+struct ZeltnerBurleySheen final : public BSDF, MxSheenParams {
+    ZeltnerBurleySheen(const MxSheenParams& params) : BSDF(), MxSheenParams(params) {}
+
+#define USE_LTC_SAMPLING 1
+
+    Color3 get_albedo(const Vec3& wo) const override
+    {
+        const float NdotV = clamp(N.dot(wo), 0.0f, 1.0f);
+        return Color3(fetch_ltc(NdotV).z);
+    }
+
+    Sample eval(const Vec3& wo, const Vec3& wi) const override
+    {
+        const Vec3 L = wi, V = wo;
+        const float NdotV = clamp(N.dot(V), 0.0f, 1.0f);
+        const Vec3 ltc = fetch_ltc(NdotV);
+
+        const Vec3 T1 = (V - N * NdotV).normalize();
```

Comment:
There's a danger of generating NaNs here if V = N (I've certainly hit this in other LTC-related code in production). I didn't bother to handle that in the WebGL sandbox, but I'm doing so in the MaterialX version:
https://github.com/AcademySoftwareFoundation/MaterialX/blob/main/libraries/pbrlib/genglsl/lib/mx_microfacet_sheen.glsl#L117-L133

Things get a little simpler if vectors are already in local space:
https://github.com/shill-lucasfilm/OpenPBR-viewer/blob/main/glsl/fuzz_brdf.glsl#L30-L37